packages/main/src/plugin/updater.ts

```diff
@@ -249,14 +270,24 @@ export class Updater {
   /**
    * Handler for update downloaded event.
    */
-  private onUpdateDownloaded(): void {
+  private onUpdateDownloaded(updatedDownloadedEvent: UpdateDownloadedEvent): void {
+    console.debug(`The update file has been downloaded to ${updatedDownloadedEvent.downloadedFile}`);
```

Comment:
I wanted to add it on purpose, in case where people want to track where it has been downloaded. As it was helpful on my side to be able to remove the file to launch again the steps.

As it's a debug trace, it's not displayed by default, you need to change the level